### PR TITLE
Add Dell Inspiron 5509

### PR DESCRIPTION
**Pull request by maydayv7**
- [x] Ran `tests/run.py` successfully

### CODE REVIEW

File: dell/inspiron/5509/default.nix [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+  # Essential Firmware
+  hardware.enableRedistributableFirmware = lib.mkDefault true;
+
+  # Cooling Management
+  services.thermald.enable = lib.mkDefault true;
+
+  services.xserver = {
+    # GPU Driver Setting
+    videoDrivers = lib.mkDefault [ "intel" ];
```
Comment (Mic92) [inline dell/inspiron/5509/default.nix:17]: We usually modsettings for intel these days. Any reason the intel xorg driver is required here?

Comment (maydayv7) [reply dell/inspiron/5509/default.nix]: Sorry, I had just copy-pasted it without realizing it, even my original configuration uses `modesetting`, I have changed it now!

File: dell/inspiron/5509/default.nix [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+  # Essential Firmware
+  hardware.enableRedistributableFirmware = lib.mkDefault true;
+
+  # Cooling Management
+  services.thermald.enable = lib.mkDefault true;
+
+  services.xserver = {
+    # GPU Driver Setting
+    videoDrivers = lib.mkDefault [ "modesetting" ];
```
Comment (Mic92) [inline dell/inspiron/5509/default.nix:17]: This is actually the default:

```suggestion
```

Comment (maydayv7) [reply dell/inspiron/5509/default.nix]: Okay, I've removed it, thanks!

**Pull request merged**